### PR TITLE
fixed linear beam3d mass matrix

### PR DESCRIPTION
**Pull request by KlausBSautter**
hi I noticed that the linear beam 3d needs its own mass matrix calculation. After we inherited the elements I forgot that.

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp
Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp:110]: Strange, it does not heritage the function from the base?

**Review by AndreasWinterstein — APPROVED** [review]
Only small but necessary change. Does not affect any other element.

File: applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp
Comment (KlausBSautter) [reply applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp:110]: no it needs always the original system as transformation in the linear case. In the bass class I am using an updated transformation which is not initialzied in the linear element. Thus I need this change to make it work

Comment (KlausBSautter) [reply applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp:110]: This was already happening before the condensation branch was merged where I started inheriting the elements

**Comment by loumalouomega** [comment]
The thing is that if the 3D case use the same function, the best, to reduce code duplication, is to implement it directly in the base class

**Comment by KlausBSautter** [comment]
the base class is the 3dnonlinear beam. the 3dlinearbeam inherits from that and needs another mass matrix calculation. this is what I did in the pr

**Comment by loumalouomega** [comment]
Ok, just asking (as you can see I hadn't block the PR)

**Comment by KlausBSautter** [comment]
yes input is always appreciated!

**Pull request merged**
